docs/source/concepts/glossary.rst

```diff
@@ -294,22 +346,22 @@ Building applications
 Application, ledger client, integration
 =======================================
 
-**Application**, **ledger client** and **integration** are all terms for an application that sits on top of the `ledger <#ledger-daml-ledger>`__. These usually `read from the ledger <#reading-from-the-ledger>`_, `send commands <#submitting-commands-writing-to-the-ledger>`__ to the ledger, or both.
+**Application**, **ledger client** and **integration** are all terms for an application that sits on top of the `ledger <#canton-ledger>`__. These usually `read from the ledger <#reading-from-the-ledger>`_, `send commands <#submitting-commands-writing-to-the-ledger>`__ to the ledger, or both.
 
 There's a lot of information available about application development, starting with the :doc:`/app-dev/app-arch` page.
 
 Ledger API
 ==========
 
-The **Ledger API** is an API that's exposed by any `Daml ledger <#ledger-daml-ledger>`__.
-Alternative names: **Daml Ledger API** and **gRPC Ledger API** if disambiguation from other technologies is needed.
+The **Ledger API** is an API that's exposed by any `ledger <#canton-ledger>`__ on a participant node. Users access and manipulate the ledger state from the perspective of a party through the leger API.
+Alternative names: **Canton Ledger API** and **gRPC Ledger API** if disambiguation from other technologies is needed.
 See :doc:`/app-dev/ledger-api` page.
 It includes the following :doc:`services </app-dev/services>`.
 
 Command submission service
 --------------------------
 
-Use the **command submission service** to `submit commands <#submitting-commands-writing-to-the-ledger>`__ - either create commands or exercise commands - to the `ledger <#ledger-daml-ledger>`__. See :ref:`command-submission-service`.
+Use the **command submission service** to `submit commands <#submitting-commands-writing-to-the-ledger>`__ - either create commands or exercise commands - to the `ledger <#canton-ledger>`__. See :ref:`command-submission-service`.
```

Comment:
You’re linking everything to Canton now. Do we just ignore that vmbc exists?

Reply:
How do you suggest that we clarify the two?

Reply:
I haven’t kept up with our messaging for 2.0 so if you tell me this is fine, I’m happy.

So far, we had the generic concept of a ledger which was anything that exposes a gRPC ledger API and conforms to the ledger API and Canton was one implementation of that.

Reply:
This is a good catch.  I've clarified the two in the description and changed the appropriate links to point to the Daml ledger.